src/System.Memory/ref/System.Memory.cs

```diff
@@ -513,6 +513,7 @@ public static partial class MemoryMarshal
         public static System.Memory<T> AsMemory<T>(System.ReadOnlyMemory<T> memory) { throw null; }
         public static System.ReadOnlySpan<TTo> Cast<TFrom, TTo>(System.ReadOnlySpan<TFrom> span) where TFrom : struct where TTo : struct { throw null; }
         public static System.Span<TTo> Cast<TFrom, TTo>(System.Span<TFrom> span) where TFrom : struct where TTo : struct { throw null; }
+        public static System.Memory<T> CreateFromPinnedArray<T>(T[] array, int start, int length) { throw null; }
```

Comment:
You're not also removing it from Memory?  Are you doing that separately?

Reply:
I was thinking of staging the change and removing it after asp.net updates its use. Depending on how quickly they can consume new corefx packages, I might remove it earlier. https://github.com/aspnet/Common/pull/335

Reply:
I wouldn't block (stage) the change this close to the RC deadline.  They can make the update as part of the dotnet-maestro-bot PR. /c @pakrym